com.unity.render-pipelines.high-definition-config/Runtime/ShaderConfig.cs

```diff
@@ -51,32 +78,55 @@ public enum ShaderOptions
         // Edit->Render Pipeline->Generate Shader Includes
         // Probe Volumes feature must also be enabled inside of your HDRenderPipelineAsset.
         // Also uncomment in the HDRP package all ".../Experimental/Probe Volume" menu
+
+        /// <summary>Probe volume evaluation mode.</summary>
         ProbeVolumesEvaluationMode = ProbeVolumesEvaluationModes.Disabled,
+        /// <summary>Probe volume supports additive blending.</summary>
         ProbeVolumesAdditiveBlending = 1,
+        /// <summary>Probe volume filtering mode.</summary>
         ProbeVolumesBilateralFilteringMode = ProbeVolumesBilateralFilteringModes.Validity,
+        /// <summary>Probe volume encoding.</summary>
         ProbeVolumesEncodingMode = ProbeVolumesEncodingModes.SphericalHarmonicsL1,
 
+        /// <summary>Support for area lights.</summary>
         AreaLights = 1,
 
+        /// <summary>Support for barn doors.</summary>
         BarnDoor = 0
     };
 
     // Note: #define can't be use in include file in C# so we chose this way to configure both C# and hlsl
     // Changing a value in this enum Config here require to regenerate the hlsl include and recompile C# and shaders
+    /// <summary>
+    /// Project wide shader configuration options.
+    /// This class reflects the ShaderOptions enum. It's meant to be used in C# code to check current configuration.
```

Comment:
add this to a /// \<remarks> section please:
This class reflects the <see cref="ShaderOptions "/> enum. Use it in C# code to check the current configuration.